exporters/otlp/internal/envconfig/envconfig.go

```diff
@@ -59,6 +59,17 @@ func WithString(n string, fn func(string)) func(e *EnvOptionsReader) {
 	}
 }
 
+// WithBool retrieves the specified config and passes it to ConfigFn as a boolean.
+func WithBool(n string, fn func(bool)) func(e *EnvOptionsReader) {
+	return func(e *EnvOptionsReader) {
+		if v, ok := e.GetEnvValue(n); ok {
+			if b, err := strconv.ParseBool(v); err == nil {
```

Comment:
Not sure we can use this.  [Boolean env values](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#boolean-value) MUST treat case-insensitive "true" as `true` and everything else as `false`.

Reply:
used a ToLower comparison with `"true"`